plugins/ocp/ocp-nvme.h

```diff
@@ -17,9 +17,9 @@ PLUGIN(NAME("ocp", "OCP cloud SSD extensions", NVME_VERSION),
 	COMMAND_LIST(
 		ENTRY("smart-add-log", "Retrieve extended SMART Information", ocp_smart_add_log)
 		ENTRY("latency-monitor-log", "Get Latency Monitor Log Page",
-		      ocp_latency_monitor_log)
```

Comment:
Seems the original code indentation correct as described by Linux kernel coding style as below.
https://docs.kernel.org/process/coding-style.html#breaking-long-lines-and-strings
```
2) Breaking long lines and strings
...
Descendants are always substantially shorter than the parent and are placed substantially to the right. A very commonly used style is to align descendants to a function open parenthesis.
```

Reply:
Let's got with the same style as we have in `nvme-builtin.h`, so one command per line even though the lines get a bit longer.